config/configgrpc/configgrpc.go

```diff
@@ -73,6 +73,14 @@ type GRPCClientSettings struct {
 	// (https://godoc.org/google.golang.org/grpc#WithKeepaliveParams).
 	Keepalive *KeepaliveClientConfig `mapstructure:"keepalive"`
 
+	// The WriteBufferSize for client gRPC. See grpc.WithReadBufferSize
+	// (https://godoc.org/google.golang.org/grpc#WithReadBufferSize).
+	ReadBufferSize int `mapstructure:"read_buffer_size"`
+
+	// The WriteBufferSize for client gRPC. See grpc.WithWriteBufferSize
+	// (https://godoc.org/google.golang.org/grpc#WithWriteBufferSize).
+	WriteBufferSize int `mapstructure:"write_buffer_size"`
```

Comment:
Do we want to expose these as options or just hard-code some reasonable value?

Reply:
Would prefer to be able to tune them. Because if user has large batches like 1000+ spans a better value is 1M instead of 512K also some pull protocols may need to tune the read.

Reply:
I think it is fine to make this configurable, at the minimum it is very useful for our own experiments.
BTW, I had the exact same issue with my [WebSocket implementation](https://github.com/open-telemetry/opentelemetry-collector/pull/1195/files) yesterday. :-) The default in gorilla/websocket is 4K which is very small for large batches. After experimentation I bumped precisely to 512K to get decent throughput. Anything larger than that has diminishing returns.